NuGet.config.orig

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <!--  Begin: Package sources from dotnet-aspnetcore -->
+    <!--  End: Package sources from dotnet-aspnetcore -->
+    <!--  Begin: Package sources from dotnet-msbuild -->
+    <!--  End: Package sources from dotnet-msbuild -->
+    <!--  Begin: Package sources from dotnet-roslyn-analyzers -->
+<<<<<<< HEAD
+    <add key="darc-pub-dotnet-roslyn-analyzers-3c1d0c4-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-roslyn-analyzers-3c1d0c48-1/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-roslyn-analyzers-3c1d0c4" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-roslyn-analyzers-3c1d0c48/nuget/v3/index.json" />
+=======
+    <add key="darc-pub-dotnet-roslyn-analyzers-755ceda" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-roslyn-analyzers-755ceda7/nuget/v3/index.json" />
+>>>>>>> upstream/release/5.0.2xx
+    <!--  End: Package sources from dotnet-roslyn-analyzers -->
```

Comment:
@sfoslund I noticed that there are some unresolved conflicts in the submitted file. Could you help to take a look?

Reply:
Thanks for noticing this Jason, in general you can delete any .orig files, they were checked in by mistake.